allensdk/brain_observatory/behavior/swdb/save_extended_stimulus_presentations_df.py

```diff
@@ -159,7 +159,7 @@ def get_extended_stimulus_presentations(session):
     # Average running speed on each flash
     flash_running_speed = intermediate_df.apply(
         lambda row: trace_average(
-            session.running_speed.values,
+            session.running_speed.speed,
```

Comment:
This is probably right, but is getting into "I have never touched this and don't know about this" territory.

Reply:
Erant thought:

The stuff in allensdk.brain_observatory.behavior.swdb is for visual behavior and not for old brain observatory stuff right?

http://alleninstitute.github.io/AllenSDK/brain_observatory.html#

Reply:
That link sure looks like Visual Coding stuff.  Are you concerned the change will break Visual Coding analysis?

Reply:
see slack/vba/sdk thread. I'm going to revert the change to this file and move on. It sounds like it is probably 2 years old and not used. It isn't referenced by VBA at all.